projects/api/src/utils/applyMiddleware.ts

```diff
@@ -0,0 +1,9 @@
+import {Router} from 'express'
+
+type Wrapper = ((router: Router) => void)
+
+export const applyMiddleware = (middleware: Wrapper[], router: Router) => {
+    for (const m of middleware) {
```

Comment:
No type for ```m``` ?

Reply:
Since middleware have type Wrapper[], m have Wrapper type by default